src/components/TodoTasks.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+import { TodoList } from '../interfaces/todo'
+import TodoTask from './TodoTask';
+
+interface Props {
+  todoList: TodoList;
+};
+
+const TodoTasks = ({ todoList }: Props) => {
+  const [todoTask, setTodoTask] = useState<TodoList>([]);
+
+  return (
+    <div className="task">
+      {todoList.map((todoTask) => {
+        return (
+          <TodoTask todoTask={todoTask} />
```

Comment:
Please add the key attribute here. React requires "key" prop to be at the Component declaration level rather than inside the iterated component.
You can use either idx as before of  todoTask.id for the value.